lib/mcb/commands/users/revoke.rb

```diff
@@ -0,0 +1,12 @@
+summary 'Remove a user from an organisation/provider in the DB.'
+param :id_or_email_or_sign_in_id
+param :provider_code, transform: ->(code) { code.upcase }
+usage 'revoke <id or email or sign-in id> <provider_code>'
```

Comment:
```suggestion
usage 'revoke <user id or email or sign-in id> <provider_code>'
```

Reply:
changed to `revoke <user_id/email/sign_in_user_id> <provider_code>`